CHANGELOG.md

```diff
@@ -1,3 +1,236 @@
+## v0.5.0 - 2020-10-23
+
+We're happy to announce the release of Lokomotive v0.5.0 (Eurostar).
+
+This release packs new features, bug fixes, code optimizations, platform updates, security hardening, and much more.
+
+### Changes in v0.5.0
+
+#### Kubernetes Updates
+
+- Update Kubernetes version to
+  [`v1.19.3`](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md#v1193)
+  ([#1030](https://github.com/kinvolk/lokomotive/pull/1030)).
+
+#### Platform updates
+
+##### AKS
+
+- Update Kubernetes version to `1.18.8`
+  ([#1071](https://github.com/kinvolk/lokomotive/pull/1071)).
+
+##### Baremetal
+
+- Expose CNI MTU on the baremetal platform
+  ([#977](https://github.com/kinvolk/lokomotive/pull/977))
+
+#### New Components
+
+- Add component `headlamp`
+  ([#981](https://github.com/kinvolk/lokomotive/pull/981)).
+- Add component `inspektor-gadget`
+  ([#1076](https://github.com/kinvolk/lokomotive/pull/1076)).
+
+#### Component updates
+
+- Update Velero component for Packet [openebs and restic plugin support]
+  ([#881](https://github.com/kinvolk/lokomotive/pull/881)).
+- istio-operator: Update to 1.7.3
+  ([#1086](https://github.com/kinvolk/lokomotive/pull/1086)).
+
+#### Terraform Updates
+
+- Upgrade to Terraform 0.13
+  ([#824](https://github.com/kinvolk/lokomotive/pull/824)).
+
+#### Features
+
+- Add encryption to in-cluster pod traffic
```

Comment:
```suggestion
- Add encryption to in-cluster pod traffic.
```

and similar missing periods (.) elsewhere.

Reply:
@ipochi periods are added after links